README.md

```diff
@@ -32,415 +32,450 @@
 
 
 ## <a name='CustomElements'></a>Custom Elements
-* A custom element, a.k.a an *autonomous custom element* , should be defined with an ES6 class extending `HTMLElement`.
+* A custom element, a.k.a an *autonomous custom element*, should be defined with an ES6 class extending `HTMLElement`.
 * The name of a custom element *must* contain a dash (-) character, this is to prevent clashes with future built-in elements.
 * A custom element must be registered with the `CustomElementRegistry`.
 
-	```javascript
-	class XElement extends HTMLElement {
-		static get observedAttributes() {/* use only when necessary */}
-		
-		constructor() {
-			super();
-		}
-		
-		connectedCallback() {/* use only when necessary */}
-		disconnectedCallback() {/* use only when necessary */}
-		adoptedCallback() {/* use only when necessary */}
-		attributeChangedCallback() {/* use only when necessary */}
+```javascript
+class MyCustomElement extends HTMLElement {
+	static get observedAttributes() { /* use only when necessary */ }
+	
+	constructor() {
+		super();
+		/* use only when necessary */
 	}
-	// an autonomous custom element
-	window.customElements.define("x-element", XElement);
-	```
+	
+	connectedCallback() { /* use only when necessary */ }
+	disconnectedCallback() { /* use only when necessary */ }
+	adoptedCallback() { /* use only when necessary */ }
+	attributeChangedCallback() { /* use only when necessary */ }
+}
+// an autonomous custom element
+window.customElements.define("my-custom-element", MyCustomElement);
+```
 * An autonomous custom element applies the defined behaviour to a custom tag.
 
-	```html
-	<x-element></x-element>
-	```
+```html
+<my-custom-element></my-custom-element>
+```
 
 ### <a name='Constructor'></a>Constructor
```

Comment:
Assuming you mean "Constructor": it is because this is the _word_ constructor, and not the _keyword_ `constructor`. You could use the keyword, but then it should not be capitalized.